sites/default/menu_data.json

```diff
@@ -590,7 +590,7 @@
             "children" : [ ],
             "requirement" : 0
         }, { 
-            "label" : "Appt-Enc",
+            "label" : "Appointments and Encounters",
```

Comment:
This one is really a financial report....  Question is, what should we view it as doing?  Appointments vs. Encounters?  This name is good enough until the real reports come in I suppose!